### ISSUE TITLE
Li-Mak test for residual from arch model

### ISSUE THREAD

**Issue by holder-fo** [bashtage/arch#405]
Hi, proffessor.
Is there any plan to add Li-Mak test for residual from arch model. I only found this test usable in R [WeightedPortTest](https://github.com/cran/WeightedPortTest), hope you can add it in your work, it would be helpful for checking ARCH effect of residual.

**Comment by bashtage** [comment]
I'm not sure this is a practical test in practice since as far as I can tell it is only designed for ARCH-type models and otherwise it requires model scores.

**Comment by holder-fo** [comment]
Yes, it is only designed for ARCH-type models and it is recommanded for checking residual from ARCH-type models. That's why i ask for your help. Or is it any alternative test to check ARCH effect of residual you can recommand to me? I am thinking how to make sure my SARIMA-GARCH model are enough. Hope you can help me.
